File: go-sdk/pkg/models/operations/getroomsroomidactiveusers.go
package operations

import (
	"openapi/pkg/models/shared"
)

type GetRoomsRoomIDActiveUsersPathParams struct {
	RoomID string `pathParam:"style=simple,explode=false,name=roomId"`
}

type GetRoomsRoomIDActiveUsersRequest struct {
	PathParams GetRoomsRoomIDActiveUsersPathParams
}

type GetRoomsRoomIDActiveUsersResponse struct {
	ActiveUsersResponse *shared.ActiveUsersResponse
	ContentType         string
	Error               *shared.Error
	StatusCode          int64
}

File: typescript-sdk/src/sdk/models/operations/getroomsroomidactiveusers.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetRoomsRoomIdActiveUsersPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=roomId" })
  roomId: string;
}


export class GetRoomsRoomIdActiveUsersRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetRoomsRoomIdActiveUsersPathParams;
}


export class GetRoomsRoomIdActiveUsersResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  activeUsersResponse?: shared.ActiveUsersResponse;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  error?: shared.ErrorT;

  @SpeakeasyMetadata()
  statusCode: number;
}
